src/elements/content-explorer/__tests__/ContentExplorer-test.js

```diff
@@ -42,16 +43,97 @@ describe('elements/content-explorer/ContentExplorer', () => {
     });
 
     describe('changeViewMode()', () => {
+        let wrapper;
+        beforeEach(() => {
+            wrapper = getWrapper();
+        });
+
         test('should initially be list view', () => {
-            const wrapper = getWrapper();
             expect(wrapper.state('viewMode')).toBe(VIEW_MODE_LIST);
         });
 
         test('should change to grid view', () => {
-            const wrapper = getWrapper();
             const instance = wrapper.instance();
             instance.changeViewMode(VIEW_MODE_GRID);
             expect(wrapper.state('viewMode')).toBe(VIEW_MODE_GRID);
         });
     });
+
+    describe('makeThumbnailUrl()', () => {
+        const template = 'baseUrl/{+asset_path}';
+        const token = 'token';
+        let instance;
+        beforeEach(() => {
+            instance = getWrapper().instance();
+        });
+
+        test('should properly make thumbnail url with png', () => {
+            expect(instance.makeThumbnailUrl('png', template, token)).toBe('baseUrl/1.png?access_token=token');
```

Comment:
Not sure if the `toBe` should be hardcoded like it is, or if it should be something like <pre>.toBe(\`${baseUrl}/1.png?access_token=${token}\`)</pre> 
Not sure which is the right philosophy for this type of test.

Reply:
Both are acceptable but I personally would prefer to make that connection between the input and the expectation so that if the inputs change later the tests properly reflect it